doc/source/contributing_code.rst

```diff
@@ -202,39 +199,77 @@ This sets the number of files to 4096. However, this command only applies to you
 Building MDAnalysis
 -------------------
 
-Make sure that you have :ref:`cloned the repository <forking-code-repo>`  
-and activated your virtual environment. First we need to install dependencies. If you're using conda, you'll need a mix of conda and pip installations:
+With conda
+----------
+
+.. note::
+    Make sure that you have :ref:`cloned the repository <forking-code-repo>`  
+    and activated your virtual environment with :code:`conda activate mdanalysis-dev`.
+    
+First we need to install dependencies. You'll need a mix of conda and pip installations:
 
     .. code-block:: bash
 
         conda install -c biobuilds -c conda-forge \
             biopython chemfiles clustalw==2.1 codecov cython \
             griddataformats gsd hypothesis "joblib>=0.12" \
             matplotlib mmtf-python mock netcdf4 networkx \
-            "numpy>=1.17.3" psutil pytest scikit-learn scipy \
-            "seaborn>=0.7.0,<0.9" sphinx==1.8.5 "tidynamics>=1.0.0" \
-            "tqdm>=4.43.0"
-
-        # if using conda with python 3.7 or 3.8, also run
-        conda install -c conda-forge parmed
-
-        # if using conda with other versions of python, also run
-        pip install parmed
+            "numpy>=1.18.0" psutil pytest scikit-learn scipy \
+            "seaborn>=0.7.0" sphinx "tidynamics>=1.0.0" \
+            "tqdm>=4.43.0" parmed
```

Comment:
@lilyminium thoughts on also adding openmm and rdkit here?

I'm kinda reluctant to get folks to pull down openmm without the cuda shim though...

Reply:
I have nothing against rdkit but would be mostly against installing openmm unless absolutely needed, which surely the user would install if they were using the converter function to have an OpenMM object to convert. Seems quite big

Reply:
Honestly, even parmed is probably unnecessary here?

Reply:
We could drop all the converter dependencies and note it somewhere in text?

Reply:
Sure, I don't think the text is even super essential though -- you need the library installed to convert to MDAnalysis, and the converters from MDAnalysis have their own error messages telling you to install them.

Reply:
Sounds good - let's remove parmed. Folks usually won't play around much with converters unless they know what they are doing anyways.

Reply:
Sounds good. I'll push the changes tonight.